x11-wm/page/page-1.9.9-r5.ebuild

```
@@ -3,11 +3,9 @@
 
 EAPI=8
 
-inherit autotools
-
 DESCRIPTION="A mouse friendly tiling window manager"
 HOMEPAGE="https://github.com/gschwind/page"
-SRC_URI="http://www.hzog.net/pub/${PN}-1.9.9-r1.tar.gz"
+SRC_URI="https://github.com/gschwind/page/releases/download/1.9.9-r5/${PN}-1.9.9-r5.tar.gz"
```

You are of course free to version your software in any way you want. I just want to point out that using a trailing -r<num> clashes a little bit with the concept of Gentoo's revisions, which are meant to signal, simply speaking, ebuild only changes. Personally I would probably use letters in this case, so you have 1.9.9a → 1.9.9b or maybe even add another version level 1.9.9.1 → 1.9.9.5.

Besides that, you could make this easier maintainable by

```suggestion
SRC_URI="https://github.com/gschwind/page/releases/download/${PVR}/${PF}.tar.gz"
```

---

Thanks for feed backs

I did updated ebuild and manifest, I will just follow a simpler versioning scheme.

Best regards